pkg/storage/postgresql/tickets_test.go

```diff
@@ -4,17 +4,113 @@ Copyright 2023 Adevinta
 
 package postgresql
 
-//import (
-//	"github.com/adevinta/vulcan-tracker/pkg/testutils"
-//	"testing"
-//)
-//
-//func TestDB_GetFindingTicket(t *testing.T) {
-//
-//	dbName := testutils.DBNameForFunc(1)
-//	dsn, err := testutils.CreateTestDatabase(dbName)
-//	if err != nil {
-//		t.Fatal(err)
-//	}
-//
-//}
+import (
+	"database/sql"
+	"testing"
+
+	"github.com/adevinta/vulcan-tracker/pkg/model"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/labstack/gommon/log"
+)
+
+var (
+	baseModelFieldNames = []string{"ID"}
+	ignoreFieldsTeam    = cmpopts.IgnoreFields(model.FindingTicket{}, baseModelFieldNames...)
+)
+
+func TestGetFindingTicket(t *testing.T) {
+
+	testStore, err := PrepareDatabaseLocal("../../../testdata/fixtures", NewDB)
```

Comment:
Also not a big deal but I would remove the first empty line above this one.

Reply:
Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034